packages/modules/itwin-viewer-react/src/services/iModel/ViewCreator.ts

```diff
@@ -274,6 +279,54 @@ export class ViewCreator {
     }
   }
 
+  /**
+   * Generate a default viewState for a blank connection
+   * @param iModel
+   * @param viewStateOptions
+   */
+  public static createBlankViewState = (
+    iModel: BlankConnection,
+    viewStateOptions?: BlankConnectionViewState
+  ) => {
+    const ext = iModel.projectExtents;
+    const viewState = SpatialViewState.createBlank(
+      iModel,
+      ext.low,
+      ext.high.minus(ext.low)
+    );
+
+    const allow3dManipulations =
+      viewStateOptions?.setAllow3dManipulations !== undefined
+        ? viewStateOptions?.setAllow3dManipulations
+        : true;
+
+    viewState.setAllow3dManipulations(allow3dManipulations);
+
+    const viewStateLookAt = viewStateOptions?.lookAt || {
+      eyePoint: new Point3d(15, 15, 15),
+      targetPoint: new Point3d(0, 0, 0),
+      upVector: new Vector3d(0, 0, 1),
+    };
+
+    viewState.lookAt(
+      viewStateLookAt.eyePoint,
+      viewStateLookAt.targetPoint,
+      viewStateLookAt.upVector,
+      viewStateLookAt.newExtents,
+      viewStateLookAt.frontDistance,
+      viewStateLookAt.backDistance,
+      viewStateLookAt.opts
+    );
+
+    viewState.displayStyle.backgroundColor =
+      viewStateOptions?.displayStyle?.backgroundColor ??
+      ColorDef.fromString(UiFramework.getColorTheme());
```

Comment:
whats the expected values for this supposed to be? i was assuming "light" or 'dark" but idk if thats right in this context
got the color theme in my sample viewer and it returned "SYSTEM_PREFERRED". Maybe just default the bg color to blue or something?

Reply:
this is not what i expected. I changed it to default to white (which is what they do in the sample showcase)

Reply:
sounds like theyre not doing it right... but this looks good now